Library/OcCryptoLib/X64/sha512-avx.nasm

```diff
@@ -0,0 +1,532 @@
+; @file
+; Copyright (C) 2021, ISP RAS. All rights reserved.
```

Comment:
This one is based on Intel code even if modified, so better to have their copyrights as well.